src/coreclr/src/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.CoreCLR.cs

```diff
@@ -606,7 +606,7 @@ public static unsafe int QueryInterface(IntPtr pUnk, ref Guid iid, out IntPtr pp
             fixed (Guid* pIID = &iid)
             fixed (IntPtr* p = &ppv)
             {
-                return ((delegate * stdcall <IntPtr, Guid*, IntPtr*, int>)(*(*(void***)pUnk + 0 /* IUnknown.QueryInterface slot */)))(pUnk, pIID, p);
```

Comment:
Unrelated to this PR, and purely style, but should we standardize on `delegate*` instead of `delegate *`? The latter looks much more like multiplication, and the former is generally what we use for other pointers.

Reply:
Fixed. Another style question is whether to have spaces between other parts: 
- `delegate* <void>`
- `delegate* unmanaged <void>`
- `delegate* unmanaged[Cdecl] <void>`
vs.
- `delegate*<void>`.
- `delegate*unmanaged<void>` or `delegate* unmanaged<void>`
- `delegate*unmanaged[Cdecl]<void>` or `delegate* unmanaged[Cdecl]<void>`

Reply:
Personally I prefer the latter, without the spaces, but I'd be ok with either.  I'd also be ok with a space after the `delegate*` but with no spaces in the remaining utterance.

Reply:
I've been using `delegate*<void>` and `delegate* unmanaged<void>` in my own code, I think its what is most easily readable overall.
* That is, only add a space if a calling convention is specified.

Reply:
Do we know what the Roslyn code formatter will be doing by default?

Reply:
I like both all spaces `delegate* unmanaged[Cdecl] <void>` or no spaces `delegate*unmanaged<void>`. Fewer characters is better tie breaker tells me to prefer the latter.  

Space just before `unmanaged` feels unbalanced to me.

cc @333fred @AaronRobinsonMSFT

Reply:
I don't like the space before the `<` because that isn't how "generics" are typically written today (which is the closest, syntactically speaking).

That is you write `T* more` and `List<T>`, not `T*more` and not `List <T>`. So `delegate*<int, void>` and `delegate* managed<int, void>` (in my mind), best match the existing coding conventions.

Reply:
The formatter is going to prefer `delegate* unmanaged[Cdecl]<void>` and `delegate*<void>`.

Reply:
>  write `T* more`

`more` is variable name in this case. It is not part of the variable type. I do not see as similar to the `delegate*` case where `unmanaged` is part of the variable type.

I do not have a strong opinion on this. I am going to switch to the Roslyn formatter style.

Reply:
That's good because there's currently no configuration setting for the spaces around the calling convention 🙂